gensim/models/phrases.py

```diff
@@ -133,19 +133,30 @@ def __init__(self, sentences=None, min_count=5, threshold=10.0,
         `delimiter` is the glue character used to join collocation tokens, and
         should be a byte string (e.g. b'_').
 
+        `recode_to_utf8` is an optional parameter to tokenize the input words into
+        utf8 encoding. For faster Phrases, you may use `recode_to_utf8=False` if you have
+        larger RAM for this memory-time tradeoff.
+
         """
         if min_count <= 0:
-            raise ValueError("min_count should be at least 1")
+            min_count = 1
```

Comment:
What are these two conditions for? I see no reason for them, so I'd suggest removing these (`min_count`, `threshold`).